server/dns_server.ml

```diff
@@ -466,13 +466,12 @@ module Notification = struct
         | Error _ -> ns
         | Ok prim -> Domain_name.Host_set.remove prim ns
       in
-      (* TODO AAAA records / use lookup_glue? *)
```

Comment:
we still don't handle `AAAA` here, correct?

Reply:
true, i pushed another commit to retain the TODO (but remove the `use lookup_glue`)